tests/interlocking_component/conftest.py

```diff
@@ -137,10 +137,10 @@ def get_gg_count() -> int:
 
 @pytest.fixture
 def route_controller(
-    configured_souc: SimulationObjectUpdatingComponent, mock_logger: Logger
+    configured_souc: SimulationObjectUpdatingComponent, event_bus: EventBus
```

Comment:
Shouldn't this be different?
```suggestion
    configured_souc: SimulationObjectUpdatingComponent, mock_event_bus: EventBus
```

Reply:
Also apply any changes two lines down.